src/backend/src/routes/tasks.routes.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import { body } from 'express-validator';
+import TasksController from '../controllers/tasks.controllers';
+import { isTaskStatus, isTaskPriority } from '../utils/tasks.utils';
+import { validateInputs } from '../utils/utils';
+import { intMinZero } from '../utils/validation.utils';
+
+const tasksRouter = express.Router();
+
+tasksRouter.post(
+  '/',
+  intMinZero(body('wbsElementId')),
```

Comment:
this may or may not be true but I think you might need to have this be wbsNumber because I think the "shared" type for wbs element only has one id and i dont know if it uses the project/wp id or the wbselementid from the database if that makes sense

Reply:
should we then do /:wbsNum in the query instead thats how we do it elsewhere

Reply:
yes